Sources/SwiftDocCPluginUtilities/PluginFlags/SkipSynthesizedSymbolsFlag.swift

```diff
@@ -0,0 +1,26 @@
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+
+extension PluginFlag {
+    /// Excludes synthesized symbols from the generated documentation.
+    ///
+    /// If `--skip-synthesized-symbols` is found, pages for synthesized symbols
+    /// will not be generated, reducing the archive file size by not emitting any `view-implementations`
+    /// folder.
+    static let skipSynthesizedSymbols = PluginFlag(
+        parsedValues: [
+            "--skip-synthesized-symbols"
```

Comment:
I think we should land this as an explicitly experimental feature to start to see how it impacts existing workflows before fully recommending it. For example, I expect existing DocC link completion strategies will now offer completions that aren't included in the built output when this flag is given.

```suggestion
            "--experimental-skip-synthesized-symbols"
```

Reply:
Makes sense, I changed the naming to ` "--experimental-skip-synthesized-symbols"`